stacks-common/src/util/secp256k1.rs

```diff
@@ -156,6 +157,7 @@ impl From<&WSTSSignature> for SchnorrSignature {
         SchnorrSignature(buf)
     }
 }
+*/
 
 impl Secp256k1PublicKey {
```

Comment:
Ideally we'd also move away from other `libsecp256k1` wrappers.  `p256k1` wraps everything, including public keys, ecdsa/schnorr sigs, etc.